stumpy/scrump.py

```diff
@@ -579,14 +635,21 @@ def prescrump(T_A, m, T_B=None, s=None, normalize=True, p=2.0, k=1):
     )
 
     if k == 1:
-        return P.flatten().astype(np.float64), I.flatten().astype(np.int64)
+        return (P.flatten().astype(np.float64), I.flatten().astype(np.int64))
```

Comment:
Why did you add extra parantheses?

Reply:
My bad. I was doing some investigation. I did undo changes but forgot to remove the parantheses added automatically by `black`. I will fix it.